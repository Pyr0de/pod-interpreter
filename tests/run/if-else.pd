let a = true;

if true {
	print "hello";
}

if a {
	print "hello2";
}

if false || true {
	print "hello3"
}

if !a {
	print "error";
}else {
	print "else";
}

if a {
	print "hello4";
}else {
	print "error2";
}

if !a {
	print "error3";
}else if a {
	print "else if";
}

if !a {
	print "error4"
}else if !a {
	print "error5"
}else {
	print "else2"
}
