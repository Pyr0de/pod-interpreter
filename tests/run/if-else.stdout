ExitCode: 0
hello
hello2
hello3
else
hello4
else if
else2
